docs/content/docs/get-started/react-tutorial.md

```diff
@@ -0,0 +1,270 @@
+---
+title: 'Tutorial: Create a Fluid Framework application with React'
```

Comment:
Is this the title that will show up on the left nav? Won't it be really long?

Reply:
Yes. That title appears in the Table of Contents on the left. It is long relative to the width of the ToC bar. But it is not too long semantically. Can the ToC be widened in Hugo? 
![image](https://user-images.githubusercontent.com/9088056/111818696-b2bdf000-889c-11eb-841c-0f2da4f7376c.png)